portal/src/components/Collections/Collection/Collection.component.js

```diff
@@ -1,16 +1,14 @@
-import _ from 'lodash';
-import BreadCrumbs from '~/components/BreadCrumbs';
-import EditableContent from '~/components/EditableContent';
-import DirectSearch from '~/components/FilterCategories/DirectSearch';
-import ShareCollection from '~/components/Popup/ShareCollection';
-import DeleteCollection from '~/components/Popup/DeleteCollection';
-import { validateHREF } from '~/components/_helpers';
-import SwitchInput from '~/components/switch-input'
-import { PublishStatus} from "~/utils";
-
+import _ from "lodash";
+import BreadCrumbs from "~/components/BreadCrumbs";
+import EditableContent from "~/components/EditableContent";
+import ShareCollection from "~/components/Popup/ShareCollection";
+import DeleteCollection from "~/components/Popup/DeleteCollection";
+import { validateHREF } from "~/components/_helpers";
+import SwitchInput from "~/components/switch-input";
+import { PublishStatus } from "~/utils";
 
 export default {
-  name: 'collection',
+  name: "collection",
```

Comment:
We werken met single quotes in frontend

Reply:
Zijn er nog meer 'afspraken'?

Reply:
2 spaces o.a. Heb daar zelf een broertje dood aan, maar ja.